src/Components/Artwork/Details.tsx

```diff
@@ -150,16 +183,23 @@ const HighDemandInfo = () => {
 
 const NBSP = " "
 
-const SaleMessage: React.FC<DetailsProps> = ({
-  artwork: { sale, sale_message, sale_artwork },
-}) => {
+const SaleMessage: React.FC<SaleMessageProps> = props => {
+  const {
+    artwork: { sale, sale_message, sale_artwork, collectorSignals },
+    showActivePartnerOfferLine,
+  } = props
```

Comment:
Out of scope of this PR, but something I would like to investigate: In order to query auction collector signals we have to query the sale/sale artwork. If we also request these fields on the artwork, does it add an unnecessary query, or do data loaders understand and skip redundant fetches?